src/components/Modal/style.css

```diff
@@ -45,7 +45,6 @@
   /* Relative positioning so overflow affects absolute children */
   position: relative;
   overflow: auto;
-  margin: auto;
```

Comment:
This will affect the style of the whole SDK, not only the demo app. Would this change break the layout of the host app?

Reply:
Good point. 

As discussed, I've realised on double checking it that the fix for CX-3871 in #755 has resolved this regression issue also. The true source of the regressions was the style added in work done for Previewer feature, which that PR fixes. Will close this PR.